book/src/object/ownership.md

```diff
@@ -0,0 +1,37 @@
+# Ownership
+
+Sui introduces four distinct ownership types for objects: single owner, shared state, immutable shared state, and object-owner. Each model offers unique characteristics and suits different use cases, enhancing flexibility and control in object management.
```

Comment:
I'm not sure I would highlight object ownership in the Move book since it is not really accessible at the Move layer